discovery/discovery.go

```diff
@@ -148,12 +148,12 @@ func doDiscover(app App, pre string, insecure bool) (*Endpoints, error) {
 // DiscoverEndpoints will make HTTPS requests to find the ac-discovery meta
```

Comment:
I had the same doubt. I splitted parse and meta because I was thinking that maybe rocket will report if one of the two failed and for which reason (see https://github.com/sgotti/rocket/commit/d2389180c779864750e67c6ae3b92a09a4f88e96). But if this isn't needed I'll merge the two ep inside DiscoverEndpoints and return an error only if both discovery methods have failed.